panorama_connector.py

```diff
@@ -362,6 +362,31 @@ def _add_commit_status(self, job, action_result):
 
         return phantom.APP_SUCCESS
 
+    def get_value_list(self, value):
```

Comment:
This is simply taken from [threatq_connector.py](https://github.com/splunk-soar-connectors/threatquotient/blob/c0ea58fb9dae034938b809b963af8c8242f1989b/threatq_connector.py#L156)